docs/source/monitoring_stack.rst

```diff
@@ -65,17 +65,17 @@ Install Scylla Monitoring
 
 .. code-block:: sh
 
-   wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.5.tar.gz
-   tar -xvf scylla-monitoring-3.5.tar.gz
-   cd scylla-monitoring-scylla-monitoring-3.5
+   wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.6.0.tar.gz
+   tar -xvf scylla-monitoring-3.6.0.tar.gz
```

Comment:
Question: can you create (and update) a 3.6 tag with hold the latest 3.6? 3.6.0 now, 3.6.1 later.
With this, you do not have to change this with each patch release.

Reply:
This is what #1039  is about, I think @lauranovich and @dgarcia360 will take care of it

Reply:
Not the same
I'm asking if you can release 3.6 so it will have the latest release zip
Not doc related

Reply:
do you mean that the gz file will be called latest?